IpmiFeaturePkg/Include/IpmiFeature.h

```diff
@@ -0,0 +1,64 @@
+/** @file
+  Ipmi structure defintions that are either not defined in MdePkg's Ipmi header files
+  or are extensions of existing function headers.
+
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef IPMI_FEATURE_H__
+#define IPMI_FEATURE_H__
+
+#include <IndustryStandard/Ipmi.h>
+

```

Comment:
Should probably have a #pragma pack(1) and unpack at the end

Reply:
Good catch!